### PR TITLE
Update Zenko-operator and backbeat

### PR DESCRIPTION
**Pull request by nicolas2bert**
Update Zenko operator and backbeat.

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ models:
   - SetProperties: &set_operator_properties
       properties:
         OPERATOR_IMAGE_NAME: registry.scality.com/sf-eng/zenko-operator
```
Comment (miniscruff) [inline eve/main.yml:70]: we should switch to using a different namespace for zenko operator, maybe for the next release @tcarmet

**Review by miniscruff — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e reset

**Review by lavigne958 — APPROVED** [review]

File: eve/main.yml (hunk 1 above)
Comment (miniscruff) [reply eve/main.yml:70]: after talking to Alex C there might be an issue given zenko operator is private while zenko is public, so where to place the image is an issue

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**